Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -249,4 +249,6 @@
     <string name="write_permission_needed">You need to allow write access to external storage for the emulator to work</string>
 
     <string name="load_settings">Loading Settings...</string>
+
+    <string name="external_storage_not_mounted">You need to have the external storage of your device mounted to continue using Dolphin</string>
```

Comment:
I don't see any reason to have this say "continue using Dolphin" instead of just "use Dolphin".

Also, everyone might not know what mounted means, so maybe we could make this say something like "The external storage needs to be available in order to use Dolphin"?